README.rst

```diff
@@ -44,13 +44,47 @@ You can run it from a source checkout without an explicit installation step::
   $ make
   $ ./gtimelog
 
-System requirements:
+
+Dependencies
+------------
 
 - Python (2.7 or 3.3+)
 - PyGObject
 - gobject-introspection type libraries for GTK+, Pango
 - GTK+ 3.10 or newer (3.14 or newer for best results)
 
+## PyPI dependencies (for installing using `pip`)
+
+- python-pip
+- pkg-config
+- libcairo-dev
+- libgirepository1.0-dev
+
+## runtime dependencies
```

Comment:
This looks like a Markdown subheading.  README.rst is a ReStructuredText file and different levels of headings are indicated by different underline characters (doesn't matter which, the ReST parser assigns levels to them in the order they're first used).